libs/design/src/molecules/media-gallery/media-renderer/media-renderer.component.ts

```diff
@@ -0,0 +1,73 @@
+import {
+	Component,
+	OnInit,
+	ComponentFactoryResolver,
+	Input,
+	Type,
+	ViewChild,
+	ViewContainerRef,
+	TemplateRef,
+	ChangeDetectionStrategy,
+} from '@angular/core';
+import { DaffMediaGalleryComponent } from '../media-gallery.component';
+import { DaffMediaGalleryRegistry } from '../registry/media-gallery.registry';
+
+@Component({
+	selector: 'daff-media-renderer',
+	templateUrl: './media-renderer.component.html',
+	changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class DaffMediaRendererComponent implements OnInit {
+	@Input() component: Type<unknown>;
+
+	constructor(
+		private componentFactoryResolver: ComponentFactoryResolver,
+		private registry: DaffMediaGalleryRegistry,
+		private gallery: DaffMediaGalleryComponent
+	) {}
+
+	@ViewChild(TemplateRef, { static: true, read: ViewContainerRef })
+	slot: ViewContainerRef;
+
+	ngOnInit() {
+		this.registry.galleries$.subscribe((galleries) => {
```

Comment:
This subscription will be made every time a new gallery is instantiated. I don't think that's what you wanted?